projects/hipdnn/plugins/miopen_legacy_plugin/engines/plans/MiopenBatchnormPlanBuilder.cpp

```
@@ -292,6 +292,19 @@ bool MiopenBatchnormPlanBuilder::isApplicable(
     [[maybe_unused]] const HipdnnEnginePluginHandle& handle,
     const hipdnn_plugin::IGraph& opGraph) const
 {
+    auto hasFloatComputeDataType = [](const auto& node) {
+        return node->computeDataType() == hipdnn_sdk::data_objects::DataType::FLOAT;
+    };
+
+    if(opGraph.nodeCount() != 0
```

Should this fail early somewhere if 0 nodes?

---

Wouldn't be a bad idea, I don't think we've got anywhere we're doing it currently though.

---

It'll fail below already I think, but I guess generally this is probably a reasonable check for the graph in the frontend to not even bother if it's 0 nodes.

---

(We don't really want to add this to the backend generally since we want to eventually support graphs we don't understand yet to flow through the system).